partiql-eval/src/main/kotlin/org/partiql/eval/PartiQLEngine.kt

```diff
@@ -31,4 +35,9 @@ public interface PartiQLEngine {
         @JvmStatic
         fun default() = PartiQLEngineBuilder().build()
     }
+
+    public class Session @OptIn(PartiQLFunctionExperimental::class) constructor(
+        var bindings: MutableMap<String, ConnectorBindings> = mutableMapOf(),
+        val functions: MutableMap<String, List<PartiQLFunction>> = mutableMapOf()
```

Comment:
```suggestion
        val bindings: Map<String, ConnectorBindings> = emptyMap(),
        val functions: Map<String, List<PartiQLFunction>> = emptyMap()
```

If we don't intend on modifying the maps, can we make these immutable?

Reply:
Sorry missed those. The original intention was to construct a builder interface for session. But you are right for now we don't need the mutable map.